### PR TITLE
Change full UI

### PR DESCRIPTION
**Pull request by sarkarBetaV**
1. changed all the necessary files.
2. fixed all the linters
3. made the website responsive for both mobile and desktop

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,100 +1,739 @@
...
+  border-radius: 8px;
+  padding: 12px;
+  align-items: center;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
   font-weight: 500;
   line-height: 24px;
+  margin: 0.9375rem 1rem 1.75rem 1rem;
+  transition: background-color 0.3s ease;
```
Comment (tugsem) [inline style.css:242]: - [ ] Please set the background of this class to white to match the figma design and better readability.

yours | figma
---|---
 |

File: index.html [hunk 2]
```diff
@@ -2,73 +2,336 @@
...
+                  alt="sidearrow"
+                />
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="bg">
+        <div class="contact-me">
+          <h2></h2>
```
Comment (tugsem) [inline index.html:299]: - [ ] Please add the `CONTACT ME` title into your contact section to match the given design:
- [ ] Also, please adjust the `get in touch` text in one line as you can see below.

yours | figma
---|---
image| image

File: style.css [hunk 3]
```diff
@@ -1,100 +1,739 @@
...
+  gap: 2.63%;
+  justify-content: flex-start;
+  margin: 2.668% 6.4%;
+}
+
+.smallbtn {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 0.55vw;
+  padding: 1% 3%;
```
Comment (tugsem) [inline style.css:216]: - [ ] Please give these more padding to match the figma design:

yours | figma
---|---
image|image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @sarkarBetaV ,👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️html semantic tags applied
✔️style almost identical to the figma design
✔️the code is well-organised
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -2,73 +2,336 @@
...
+                />
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="bg">
+        <div class="contact-me">
+          <h2>CONTACT ME</h2>
+          <p>
```
Comment (usorfaitheloho) [inline index.html:300]: As stated by the previous reviewer, you should implement the change to fit the design given. Kindly use a lower letter character for the `Contact Me` header of your contact form and use the font properties given in the design 😄

File: index.html [hunk 5]
```diff
@@ -2,73 +2,336 @@
...
+              placeholder="abc@domain.com"
+            />
+            <textarea
+              name="message"
+              id="message"
+              placeholder="Write your message here..."
+              required
+              maxlength="500"
+            ></textarea>
+            <button type="submit" class="butn">Get in touch</button>
```
Comment (usorfaitheloho) [inline index.html:330]: Please your `Get in touch` button styling should be similar to the one given in the design 😄 

Figma design :
image

Your design :

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @sarkarBetaV  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good use of  HTML&CSS best practices ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @sarkarBetaV,**

You did a really great job by making the changes requested by the previous reviewers💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**